app/models/manageiq/providers/redhat/inventory/parser.rb

```diff
@@ -0,0 +1,14 @@
+class ManageIQ::Providers::Redhat::Inventory::Parser < ManagerRefresh::Inventory::Parser
+  require_nested :InfraManager
+
+  def process_inventory_collection(collection, key)
```

Comment:
e.g. https://github.com/Ladas/manageiq-providers-amazon/blob/f8615fb4c69e7485b57f4f1ffd4f6bbbc4782438/app/models/manageiq/providers/amazon/inventory/parser/cloud_manager.rb#L169-L169

so process_inventory_collection is not needed anymore